react/OrderShipping.tsx

```diff
@@ -49,6 +56,10 @@ export const OrderShippingProvider: React.FC = ({ children }) => {
   const [estimateShipping] = useMutation(EstimateShippingMutation)
   const [selectDeliveryOption] = useMutation(SelectDeliveryOptionMutation)
   const [updateSelectedAddress] = useMutation(UpdateSelectedAddressMutation)
+  const [
+    autocompletedAddress,
```

Comment:
`searchAddress`? 😛

Reply:
IMO `searchAddress` is better, but looks like a function. So...
`searchedAddress`? 😜

Reply:
`search(ed)Address` is fine, but the problem is that the API has an address type called `search`, and I'm afraid someone might confuse these two concepts.

Reply:
I think either `searchedAddress` or `autocompletedAddress` is fine

Reply:
Same. Can't think on anything less confusing than `searchedAddress` or better than `autocompletedAddress`, so guess I'm okay with the latter.